iothub/device/src/Transport/FileUploadCompletionNotification.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Client.Transport
+{
+    /// <summary>
+    /// The request payload to send to IoT Hub to notify it when a file upload is completed, whether successful or not.
+    /// </summary>
+    public class FileUploadCompletionNotification
```

Comment:
q - are these fields mandatory to be set?

Reply:
good question, let me check

Reply:
CorrelationId and Success are mandatory, the others are optional